core/templates/dev/head/player/learner_local_nav.html

```diff
@@ -1,5 +1,12 @@
 <ul class="nav navbar-nav oppia-navbar-nav navbar-right"
     ng-controller="LearnerLocalNav" style="margin-right: 0px;">
+  <li>
+    {% if username %}
+      <a ng-click="showFlagExplorationModal()" tooltip="Flag Exploration" tooltip-placement="bottom">
```

Comment:
Can you put this button last instead of first? I think it'll be the least frequent action of the three (well, hopefully).